recipes/cloud-functions-utils/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "cloud-functions-utils" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cloud-functions-utils-{{ version }}.tar.gz
+  sha256: e773da8e00c7400a42480e2bc321304aa69730c6e4b22b63ce1d8529722020d1
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<4.0
```

adding `poetry` should resolve the build error. 🙂 

```suggestion
    - pip
    - python >=3.6,<4.0
    - poetry
```